config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,15 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = sleep 2s
+# Test will find container by CID instead of name, CID must appear
+# on stdout.
+run_options_csv = --detach --tmpfs /run
```

Comment:
nit-pick, this is suppose to be a comma-separated list of arguments, but for now it won't cause any problems.